src/components/AppContent/AppContent.vue

```diff
@@ -21,17 +21,84 @@
  - along with this program. If not, see <http://www.gnu.org/licenses/>.
  -
  -->
+<docs>
+### General description
+
+This components provides a wrapper around the main app's content.
+
+Single-column layouts can just use the default slot. A resizable column
+can be added by providing content to the named slot `list`.
+
+### Examples
+
+#### Usage: Single-column content
+```vue
+<template>
+	<AppContent>
+		<h2>Single-column main content</h2>
+	</AppContent>
+</template>
+```
+
+#### Usage: Two resizable columns
+```vue
+<template>
+	<AppContent>
+		<template slot="list">
+			<div>Resizable list content</div>
+		</template>
+
+		<div>Main content</div>
+	</AppContent>
+</template>
+```
+
+#### Overriding Defaults
+The default, min and max sizes (in percent) of the resizable list column can be overridden.
+The list size must be between the min and the max width value.
+
+```
+<AppContent
+	list-size="35"          # default: 25
+	list-min-width="20"     # default: 15
+	list-max-width="45"     # default: 40
```

Comment:
```suggestion
	list-size="35"
	list-min-width="20"
	list-max-width="45"
```
These comments lead to a `String contains invalid character error` on netlify and are not necessary, as the default can be seen from the props.